packages/govern-create/helpers/etherscan-verification.ts

```diff
@@ -0,0 +1,85 @@
+import { exit } from 'process'
+import fs from 'fs'
+import { file } from 'tmp-promise'
```

Comment:
Should we name this `tmpFile` for clarity?

```suggestion
import { file as tmpFile } from 'tmp-promise'
```